### DISCUSSION TITLE
Modified Date  Field- Changing Field Format

### DISCUSSION THREAD

**Discussion by ki0kk** [joomlaboat/custom-tables#86] [General]
I want to change the format of the Change Time Field. This appears to be the PHP definition with parameters {{ FieldName("Y-m-d") }} but where should the revised info be entered? I want the format to be "3/12/2024 14:00:00".

**Comment by joomlaboat** [comment]
Hello,

Try this:

{{ FieldName("d/m/Y h:j:s") }}

**Comment by ki0kk** [comment]
Yes but where does it need to be entered?

**Comment by joomlaboat** [comment]
In the Layout:
![image](https://github.com/joomlaboat/custom-tables/assets/8064340/45d65715-aab1-455c-bb23-c15352c10aa7)

**Comment by ki0kk** [comment]
That did it. Guess I learned something that I had not pickled up on before.

TNX

**Comment by joomlaboat** [comment]
You are welcome. Also, you can find more Tag options by double clicking on them.
